api/getPageContentByPathname/index.ts

```diff
@@ -23,7 +22,7 @@ export default async function getPageContentByPathname(pathname: string): Promis
   })
 
   const json = await result.json()
-  const [pageContent] = json?.data?.pageContentCollection?.items || []
```

Comment:
I think the JSON data should be this type. Otherwise, the backend is broken. Please correct me if I am wrong.